### ISSUE TITLE
Is this project deprecated?

### ISSUE THREAD

**Issue by HeathNaylor** [jarulraj/sqlcheck-old#28]
I see that the name for the repository has changed to `sqlcheck-old`. Is this still going to be a maintained project or is it considered feature frozen/deprecated?

**Comment by jarulraj** [comment]
Yes, we are working on a more sophisticated anti-pattern checker that we will likely share as a service.

**Comment by amesas** [comment]
Only as a service?
We have been using sqlcheck in our CI/CD, and we would like to keep it as it is.
